apps/laboratory/tests/multichain/multichain-ethers-solana-siwe.spec.ts

```diff
@@ -28,6 +28,9 @@ test.beforeAll(async ({ browser }) => {
   await modalPage.load()
   await modalPage.qrCodeFlow(modalPage, walletPage)
   await modalValidator.expectConnected()
+  await modalPage.promptSiwe()
+  await walletPage.handleRequest({ accept: true })
+  await modalValidator.expectAuthenticated()
```

Comment:
This won't expect to prompt siwe because before we were filtering out the solana chain ids when trying to authenticate, as we are not doing it anymore and the OneClickAuth feature won't accept multiple namespaces, the sign message step is now needed for multichain auth.

https://github.com/reown-com/appkit/blob/main/packages/appkit/src/client.ts#L722-L724